basicMysql/register/forgot.php

```diff
@@ -0,0 +1,54 @@
+<?php
+	session_start();
+  if(isset($_SESSION['login_user'])){
+	header("location: ../../index.php");
+  }
+  else{
+    session_unset();
+  }
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Forgot-User</title>
+	<style>
+		<?php include "../../style.css" ?>
+	</style>
+
```

Comment:
We should maintain the same space, lines 13 to 19 have different spacing after that line 21 has different spaces.